src/bootstrap/src/core/download.rs

```diff
@@ -706,7 +706,7 @@ download-rustc = false
             let file_times = fs::FileTimes::new().set_accessed(now).set_modified(now);
 
             let llvm_config = llvm_root.join("bin").join(exe("llvm-config", self.build));
-            let llvm_config_file = t!(File::open(llvm_config));
+            let llvm_config_file = t!(OpenOptions::new().read(true).write(true).open(llvm_config));
```

Comment:
You could save yourself an import using `File::options()`.

Also I don't think you need `read`?

Reply:
Switched to `File::options` and removed `read()`. Thanks!